examples/remix/app/test.tsx

```diff
@@ -0,0 +1,47 @@
+import { css } from './css';
+
+export const expectInvalidPropertiesToError = css({
+  // @ts-expect-error
+  '--boop': 'boop',
+});
+
+export const expectInvalidValuesToError = css({
+  // @ts-expect-error
+  '--border-color': 'red',
+});
+
+export const expectInvalidSelectorToError = css({
+  // @ts-ignore
```

Comment:
i've had to `ts-ignore` the errors that are produced by the ts-plugin bcos i've just discovered that you cannot use `ts-expect-error` or `ts-error` with ts plugins 🙈  to get `ts-error` to work, i've had to add [a custom implementation](https://github.com/tokenami/tokenami/pull/213/files#diff-7bef4b222aff4360dd908e3307dbaa860b66429fa84d6fbda3d192b62ea96e3dR84) but `ts-expect-error` is trickier bcos you need to suppress the error but also suppress `ts-expect-error` if the line below it correctly errors. i've decided that's probably a separate task to do outside of the scope of this bugfix.